### PR TITLE
[ADD] inventory: compare packs and uom

### PR DESCRIPTION
**Pull request by Felicious**
Overview doc that compares UoM, packages, and packagings. Requested by [Aurora (auca) ](https://www.odoo.com/odoo/project/3835/tasks/4288104) to inform customers which features are suitable for their business while implementing Odoo with a BSA.

Also added section to detail the Move Entire Packages feature

[Task](https://www.odoo.com/odoo/project/3835/tasks/4288104)
Fwport: yes, all the way to master (:

### CODE REVIEW

**Comment by Felicious** [comment]
Hi @ksc-odoo ! This PR is ready for the first round of peer review. Since this is an analytical doc with few setup instructions, I'll also be following up with a product expert to ensure the concepts are explained properly

**Comment by Felicious** [comment]
Implemented suggests from AUCA and BRHE to add a column for UoM in the comparison section, as well as improve the first use case for clarity. Consulted Inventory smartclass content to come up with the third use case about fruit juice (:

**Review by ksc-odoo — APPROVED** [review]
hey @Felicious -- just finished my Peer Review of this PR. Great work! Really detailed and smooth throughout. Once you address my comments, and implement the necessary changes, feel free to move this to Tech Review. Once again -- awesome job! 👍

**Comment by Felicious** [comment]
Thank you for the peer review, @ksc-odoo ! Since this PR has been looked at by several experts, including Aurora, BRHE, and GME, I'll move this onto technical review!

**Comment by Felicious** [comment]
Hello @samueljlieber ! This PR is ready for technical review! There's no need to rush on this :blush: Merry Christmas and happy holidays! :christmas_tree:

File: content/applications/inventory_and_mrp/inventory/product_management/configure.rst [hunk 1]
```diff
@@ -1,9 +1,167 @@
-:nosearch:
+:show-content:
+:hide-page-toc:
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/product_management/configure.rst:2]: I recommend showing the _on page TOC_ (not to be confused with the es.html#document-metadata)
```suggestion
```

File: content/applications/inventory_and_mrp/inventory/product_management/configure/package.rst [hunk 2]
```diff
@@ -45,6 +45,20 @@ the :guilabel:`Operations` heading, activate the :guilabel:`Packages` feature. T
...
+
+To do that, go to :menuselection:`Inventory app --> Configuration --> Operations Types` and select
+the desired operation this feature will apply to (may have to set it for multiple).
+
+On the operation type page, in the :guilabel:`Packages` section, tick the :guilabel:`Move Entire
+Packages` checkbox.
+
+.. image:: package/move-entire.png
+   :alt: Activate the Move Entire Packages setting.
+
```
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/product_management/configure/package.rst:61]: IMO this image seems unnecessary as your written instructions are clear enough :)
```suggestion
```

File: content/applications/inventory_and_mrp/inventory/product_management/configure/package/move-entire.png
[the remark this answers is not in the document]
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/product_management/configure/package/move-entire.png:1]: removed the image, as per your suggestion!

**Comment by Felicious** [comment]
@robodoo r+

**Pull request closed without merging**